cmd/sha1sum/sha1sum.go

```diff
@@ -5,12 +5,22 @@ import (
 	"os"
 
 	"github.com/rclone/rclone/cmd"
+	"github.com/rclone/rclone/fs/config/flags"
+	"github.com/rclone/rclone/fs/hash"
 	"github.com/rclone/rclone/fs/operations"
 	"github.com/spf13/cobra"
 )
 
+var (
+	outputBase64 = false
+	downloadFlag = false
+)
+
 func init() {
 	cmd.Root.AddCommand(commandDefinition)
+	cmdFlags := commandDefinition.Flags()
+	flags.BoolVarP(cmdFlags, &outputBase64, "base64", "", outputBase64, "Output base64 encoded hashsum")
```

Comment:
It would be nice to factor the flags...

I'd move all the flags to `hashsum` and have a public `AddFlags` function which md5sum/sha1sum etc call.

You can either make the flags global variables and refer to them in md5sum/sha1sum like that or you can make a wrapper for this `operations.HashLister(context.Background(), hash.SHA1, outputBase64, downloadFlag, fsrc, os.Stdout)`

Reply:
Normally if code passes Nick's review, it works.

Reply:
As I understood it, there were 2 possible approaches.  Option 1, public flags.  Option 2, a wrapper.  I do not understand how to implement option 2.  I have implemented Option 1 in a manner largely consistent with check / cryptcheck.